src/Sulu/Bundle/ContentBundle/Preview/PreviewRenderer.php

```diff
@@ -89,6 +89,8 @@ public function render(PageBridge $content, $partial = false)
         // get controller and invoke action
         $request = new Request($query, $request, [], $cookies);
         $request->attributes->set('_controller', $content->getController());
+        $request->attributes->set('webspace', $content->getWebspaceKey());
+        $request->attributes->set('locale', $content->getLanguageCode());
```

Comment:
Or better question: Shouldn't we be setting `_sulu` with the value of a `RequestAttribute` here, as it seems to be done in the rest of the code?